README.md

```diff
@@ -191,7 +191,7 @@ Hurray! You successfully made a contribution! 🎉
   - [Aniruddha Inge](https://github.com/Aniruddha-Inge)
   - [Anirudh Eyyani](https://github.com/AnirudhEyyani)
   - [Anjali Goswami](https://github.com/anjaligoswami)
-  - [Ankit Ojha](https://github.com/ankitojha07)
+  - [Anjan Diyora](https://github.com/Anjan50)
```

Comment:
I think you did something wrong here. Please add that name back. Let us know if you need any help. 🙂